src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -4135,5 +4135,32 @@ public class Methods2
             var expectedDescription = $"void Methods1.Do(string x)";
             VerifyWithReferenceWorker(markup, MainDescription(expectedDescription));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.QuickInfo)]
+        [WorkItem(4868, "https://github.com/dotnet/roslyn/issues/4868")]
+        public void QuickInfoExceptions()
+        {
+            Test(@"
+using System;
+namespace MyNs
+{
+    class MyException1 : Exception { }
+    class MyException2 : Exception { }
+    class TestClass
+    {
+        /// <exception cref=""MyException1""></exception>
+        /// <exception cref=""T:MyNs.MyException2""></exception>
+        /// <exception cref=""System.Int32""></exception>
+        /// <exception cref=""double""></exception>
+        /// <exception cref=""Not_A_Class_And_Therefore_Not_Listed""></exception>
+        void M()
+        {
+            M$$();
+        }
+    }
+}
+",
+                Exceptions($"\r\n{WorkspacesResources.Exceptions}\r\n  MyException1\r\n  MyException2\r\n  double\r\n  int"));
+        }
```

Comment:
I pinged @brettfo that the sort order looks odd here.. He explains that we sort using fully qualified names (MyNs.MyEx, System.Double) and then symbol display uses a simplified name, thus making it look unsorted.

I think if we're taking the time to sort the list, we should ensure it looks sorted. If its going to look unsorted, we should atleast save the time used for sorting and not sort at all  :smile: